### PR TITLE
dnsdist: add documentation links to dnsdist.service

### PR DESCRIPTION
**Pull request by rubenk**

### CODE REVIEW

File: pdns/dnsdistdist/dnsdist.service.in [hunk 1]
```diff
@@ -1,5 +1,7 @@
 [Unit]
 Description=DNS Loadbalancer
+Documentation=man:dnsdist(1)
+Documentation=https://dnsdist.org
```
Comment (pieterlexis) [inline pdns/dnsdistdist/dnsdist.service.in]: the dnsdist website does not have https.

**Comment by pieterlexis** [comment]
one nit, awesome otherwise

File: pdns/dnsdistdist/dnsdist.service.in (hunk 1 above)
Comment (rubenk) [inline pdns/dnsdistdist/dnsdist.service.in]: It does, with a self-signed certificate ;)

I'll fix it up.

**Comment by rgacogne** [comment]
:+1: except for the https link :-)

**Comment by rubenk** [comment]
Fixed.

**Pull request merged**